udata/tests/api/test_datasets_api.py

```diff
@@ -1366,17 +1414,10 @@ def test_bulk_update(self):
             }
             for id in ids
         ]
-        data.append(
-            {
-                "title": faker.sentence(),
-                "description": faker.text(),
-                "url": faker.url(),
-            }
-        )
         response = self.put(url_for("api.resources", dataset=self.dataset), data)
         self.assert200(response)
         self.dataset.reload()
-        self.assertEqual(len(self.dataset.resources), 3)
+        self.assertEqual(len(self.dataset.resources), 2)
```

Comment:
We used to be able to add resources on bulk update (PUT `/api/1/resources/`) somehow? We can't anymore.